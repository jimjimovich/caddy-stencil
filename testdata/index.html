{
"title": "test json"
}
<p>Some HTML</p>