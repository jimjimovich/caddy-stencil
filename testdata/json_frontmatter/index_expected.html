<!DOCTYPE html>
<html>
    <head>
        <title>Title from JSON</title>
    </head>
    <body>
        
        <p>Then some HTML from the template</p>
        Finally, a footer from <strong>JSON</strong>
    </body>
</html>